lib/puppet/network/http/api/v3/indirection_type.rb

```diff
@@ -0,0 +1,13 @@
+class Puppet::Network::HTTP::API::V3::IndirectionType
+
+  INDIRECTION_TYPE_MAP = {
+    "certificate" => :ca,
+    "certificate_request" => :ca,
```

Comment:
@camlow325 totally unrelated to this PR but I think this ^^ explains the plural/singular discrepancy that you were asking me about the other day.